Users/toml_config.rst

```diff
@@ -0,0 +1,155 @@
+  or duplicate keys in same section.
+- key-value pairs are building blocks of a TOML document. Use key-value
+  pairs to specify rules for coala bears.
+- A table is a collection of key-value pairs. Use a table for specifying
+  a coala section.
+
+::
+
+    [cli]
+    bears = 'SpaceConsistencyBear'
+    files = 'src/*.c'
+    use_spaces = true
+
+    [invalidlinks]
+    enabled = false
+    files = [ '**/*.rst', 'README.rst']
+    ignore = 'venv/**'
+    bears = 'InvalidLinkBear'
+
+Here tables ``cli`` and ``invalidlinks`` are coala sections.
+The contents of the tables like ``bears``, ``files`` are rules
+that govern a section. In coala you will be using TOML strings,
+booleans, integers and arrays as values.
+
+Section Inheritance
+----------------------------
+coala supports section inheritance. You can define section inheritance
+explicitly by naming a section in the format ``["basesection.newsection"]``.
+Extra values can be appended to an inherited setting using the ``appends`` key.
+
+.. note::
+
+    In ``["basesection.newsection"]``, the quotes insides the square braces are
+    necessary for specifying section inheritance in TOML.
+
+
+Consider the following coafile::
+
+  [all]
+  enabled = true
+  overridable = 2
+  ignore = 'vendor1/'
+
+  ["all.section1"]
+  overridable = 3
+  appends = 'ignore'
+  ignore = 'vendor2/'
+  other = 'some_value'
+
+  ["all.section2"]
+  overridable = 4
+  ignore = 'vendor3/'
+  appends = 'ignore'
+  other = 'some_other_value'
+
+
+In the inherited sections above, ``appends`` key specifies that the value of
+``ignore`` in the derived sections must be appended with the value of
+``ignore`` key in the base section. This is the same file without section
+inheritance::
+
+  [all]
+  enabled = true
+  overridable = 2
+  ignore = 'vendor1/'
+
+  [section1]
+  enabled = true
+  overridable = 3
+  ignore = ['vendor1/', 'vendor2/']
+  other = 'some_value'
+
+  [section2]
+  enabled = true
+  overridable = 4
+  ignore = ['vendor1/', 'vendor3/']
+  other = 'some_other_value'
+
+
+Defining Aspects and Tastes
+---------------------------
+
+Aspects is an alternative way to configure coala. In this mode, we don't need
+to explicitly state list of bears, coala will choose it automatically based on
+requested aspects in configuration file. To run coala in this mode, we need to
+define `aspects`, `files`, `languages`, and optionally aspect tastes setting.
+See the following example::
+
+  [all]
+  files = '**'
+  aspects = ['aspectname1', 'AspectName2'] # case-insensitive
+  # defining an aspect's taste
+  aspectname1.aspect_taste = 80
+  # we can define subaspect taste through its parent
+  aspectname1.subaspect_taste = ['word1', 'word2', 'word3']
+
+  ['all.python']
+  files = '**.py'
+  language = 'Python'
+  # appending additional aspect
+  appends = 'all'
+  aspects = 'aspectname3'
+  # excluding certain subaspect
+  excludes = 'AspectName2Subaspect'
```

Comment:
Maybe it is useful to have like a side-by-side comparison of how the syntax is. You can start with a complex coafile in coala format and explain how it can be written in toml format because users already know how to write in coala format but not in toml.